docs/reference/reference-interface/job-config-reference.rst

```diff
@@ -161,3 +161,6 @@ The following configuration settings are supported:
       terminal that is currently being viewed in JupyterLab is inputting or outputting any data. (A
       terminal that is running a command but not being viewed or running a command with no output is
       treated as idle, since JupyterLab does not provide activity information for those case.)
+
+-  ``slurm``: Slurm cluster details may optionally be specified as for :ref:`experiments
+   <experiment-config-reference>`.
```

Comment:
Preferable to link the the slurm option specifically (might need a new tag to reference) instead of the whole of expconfig page.